bin/check_samplesheet.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
```

Comment:
I am not even reviewing with the assumption that this is all same. Let me know if that's not the case.